go-controller/pkg/ovn/gateway_init.go

```diff
@@ -155,6 +165,16 @@ func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*n
 						"stderr: %q, error: %v", gatewayRouter, proto, stderr, err)
 				}
 			}
+			if workerProtoLBMap[proto] == "" {
+				workerProtoLBMap[proto], stderr, err = util.RunOVNNbctl("--", "create",
+					"load_balancer",
+					fmt.Sprintf("external_ids:k8s-worker-lb-%s=%s", strings.ToLower(string(proto)), nodeName),
```

Comment:
note, we should start to move this name conventions to constants `k8s-worker-lb-` since they are really an API now

Reply:
right, i had made constants for the specific balancers, but not this:
```
pkg/types/const.go:	WorkerLBTCP   = "k8s-worker-lb-tcp"
pkg/types/const.go:	WorkerLBUDP   = "k8s-worker-lb-udp"
pkg/types/const.go:	WorkerLBSCTP  = "k8s-worker-lb-sctp"

```

I'll add a constant and clean this up.